cloud-watch-to-slack-testing-58006136-c30b-43d1-880c-7b36f860e7c8/deployment/index.js

```diff
@@ -153,6 +153,16 @@ function processEvent(event, callback) {
     const newState = message.detail.newEvaluationResult.complianceType;
     messageText = `${alarmName} state is now ${newState}`;
     sendMessageToSlack(messageText, callback);
+  } else if (message.source == "aws.guardduty") {
+    messageText =
```

Comment:
FYI, you can use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) in JavaScript. I personally find them easier, but your code works too.